docs/api/checkouts.rst

```diff
@@ -957,3 +957,33 @@ Exemplary Response
 .. code-block:: text
 
     STATUS: 204 No Content
+
+Last possible action during order completion is changing the order locale:
```

Comment:
Quite sure this is a cart modification, not order's, besides it gets a bit confusing when there are **2** last final actions (notes/locale).
Also they're both going through with `PATCH`, seems both of them could be merged as one example with **every** final change to the cart, before it becomes an order.